src/families/ethereum/specs.js

```diff
@@ -37,6 +38,39 @@ const ethereum: AppSpec<Transaction> = {
         );
       },
     },
+    {
+      name: "move all of ERC20 tokens to another account",
+      transaction: ({ account, siblings, bridge }) => {
+        invariant(account.subAccounts, "no erc20 tokens available");
+        invariant(
+          account.balance.gt(minimalAmount),
+          "parent balance is too low"
+        );
+        const erc20Account = sample(account.subAccounts);
```

Comment:
this can be undefined. (no token accounts yet). it's good to have an invariant instead of a js crash). also to me we should `sample(` on the token accounts filtered by `gt(0)`